### PR TITLE
[SYCL] Disable test assert_in_simultaneous_kernels due to flaky failures

### PR DESCRIPTION
**Pull request by KseniyaTikhomirova**
Signed-off-by: Tikhomirova, Kseniya <kseniya.tikhomirova@intel.com>

### CODE REVIEW

**Review by againull — DISMISSED** [review]

File: SYCL/Assert/assert_in_simultaneous_kernels.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// REQUIRES: linux
+// REQUIRES: TEMPORARY_DISABLED
```
Comment (vladimirlaz) [inline SYCL/Assert/assert_in_simultaneous_kernels.cpp]: The test was Linux specific. After replace of linux with TEMPORARY_DISABLED we miss this info.
```suggestion
// REQUIRES: linux, TEMPORARY_DISABLED
```

Comment (vladimirlaz) [inline SYCL/Assert/assert_in_simultaneous_kernels.cpp]: Please add comment with the disable reason. Also link to the issues is needed

**Review by vladimirlaz — CHANGES_REQUESTED** [review]

File: SYCL/Assert/assert_in_simultaneous_kernels.cpp (hunk 1 above)
Comment (KseniyaTikhomirova) [reply SYCL/Assert/assert_in_simultaneous_kernels.cpp]: Thank you, done

Comment (KseniyaTikhomirova) [reply SYCL/Assert/assert_in_simultaneous_kernels.cpp]: The failure was observed here: https://github.com/intel/llvm/pull/4533

**Review by vladimirlaz — APPROVED** [review]

**Pull request merged**